tests/components/fronius/test_diagnostics.py

```diff
@@ -0,0 +1,31 @@
+"""Tests for the diagnostics data provided by the KNX integration."""
+
+from syrupy import SnapshotAssertion
+
+from homeassistant.core import HomeAssistant
+
+from . import mock_responses, setup_fronius_integration
+
+from tests.components.diagnostics import get_diagnostics_for_config_entry
+from tests.test_util.aiohttp import AiohttpClientMocker
+from tests.typing import ClientSessionGenerator
+
+
+async def test_diagnostics(
+    hass: HomeAssistant,
+    hass_client: ClientSessionGenerator,
+    aioclient_mock: AiohttpClientMocker,
+    snapshot: SnapshotAssertion,
+):
```

Comment:
```suggestion
) -> None:
```

Reply:
huh, I wonder why pre-commit (mypy / ruff) didn't catch that 🤔

Reply:
Strict typing is not enforced on tests.